templates/bot/js/default/package.json

```diff
@@ -15,8 +15,8 @@
   "dependencies": {
     "@microsoft/teamsfx": "^0.1.0",
     "@microsoft/microsoft-graph-client": "^2.2.1",
-    "botbuilder": "~4.11.0",
-    "botbuilder-dialogs": "~4.11.0",
+    "botbuilder": "~4.14.0",
```

Comment:
I am a little curious on these templates using different versions, and good to see they are aligned again. Since we upgrade the version for different templates, we need to test all of them to check functionality and SSO work correctly.

Reply:
I of course have tested them all.